web/source/kmwosk.js

```diff
@@ -700,8 +700,8 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
       // TODO:  Refactor the overloading of the 'n' parameter here into separate methods.
 
       // Test for fall back to U_xxxx key id - For this first test, we refer to Unicode values.
-      if((keyName.substr(0,2) == 'U_') && (n > osk.keyCodes['K_SPACE']) &&  !(n>127 && n<160)) { // 127 - 160 refer to Unicode control codes.
-        ch=String.fromCharCode(n);
+      if((keyName.substr(0,2) == 'U_') && ((keyShiftState == 0) || (n > osk.keyCodes['K_SPACE']) &&  !(n>127 && n<160))) { // 127 - 160 refer to Unicode control codes.
+        ch=String.fromCharCode(parseInt(keyName.substr(2,6),16));
```

Comment:
How is `keyShiftState` related to this?  Also, I'm having difficulty recognizing the conditions at play here as they related to #160.  Maybe the best question to ask:  where are ` VKDictionary` entries made?